docs/api/en/core/Object3D.html

```diff
@@ -438,11 +438,22 @@ <h3>[method:null traverseAncestors]( [param:Function callback] )</h3>
 		</p>
 
 		<h3>[method:null updateMatrix]()</h3>
-		<p>Update the local transform.</p>
+		<p>Updates the local transform matrix, signals to update global transform matrix before rendering.</p>
```

Comment:
> signals to update global transform matrix before rendering

Why do you think is this information beneficial for users? IMO, it's an internal detail that does not need to be exposed.

Reply:
The idea is to make it obvious that the global transform won't be immediately updated by this method. But perhaps the old description was good enough.

Reply:
> But perhaps the old description was good enough.

I would say so since it explicitly mentioned `local transform`.